app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddTraderReferencePage.scala.html

```diff
@@ -84,12 +85,12 @@
           RadioItem(
             content = Text(messages("common.yes")),
             value = Some("true"),
-            checked = form.data.get("should-store-trader-ref").contains("true"),
+            checked = form.data.get("claim-trader-ref").nonEmpty,
             conditionalHtml = Some(textHtml)
           ),
           RadioItem(
             content = Text(messages("common.no")),
-            checked = form.data.get("should-store-trader-ref").contains("false"),
+            checked = form.data.get("claim-trader-ref").isEmpty,
```

Comment:
Will this not cause "NO"to be checked by default?

Reply:
I test this in unit test cases and by default it gives the value checked as false. if you revert it to the previous one, t works fine